install/operator/pkg/generator/assets/database/syndesis-db.yml.tmpl

```
@@ -29,6 +29,17 @@
       syndesis.io/component: syndesis-db
     name: syndesis-sampledb-config
   data:
+    delete-sample-db.sh: |
+      #!/bin/bash
+      until bash -c "psql -h 127.0.0.1 -U $POSTGRESQL_USER -q -d $POSTGRESQL_DATABASE -c 'SELECT 1'"; do
+        echo "Waiting for Postgres server..."
+        sleep 1
+      done
+      echo "***** deleting sampledb"  
+      psql <<EOF
+        DROP DATABASE sampledb;
+        DROP USER sampledb;
```

Might be a bit safer

```suggestion
        DROP DATABASE IF EXISTS sampledb;
        DROP USER IF EXISTS sampledb;
```